amitools/fs/FileName.py

```diff
@@ -74,8 +77,10 @@ def is_valid(self):
         s = self.name.get_ami_str()
         # check for invalid chars
         for c in s:
-          o = ord(c)
-          if o == ':' or o == '/':
+          # o = ord(c)
+          # if o == ':' or o == '/':
+          # FIXME: FS
+          if c == ':' or c == '/':
```

Comment:
PS: You may want to check this fix, looks to me like the original code was broken.